engine/src/main/java/org/terasology/logic/health/DoDestroyEvent.java

```diff
@@ -36,11 +34,34 @@ public EntityRef getInstigator() {
         return instigator;
     }
 
+    public String getInstigatorString() {
+        if (instigator == null) {
+            return "Unknown";
+        }
+        if (instigator.getParentPrefab() != null) {
+            String instigatorString = instigator.getParentPrefab().getName();
+            instigatorString = instigatorString.replaceAll("[A-Za-z]*:([A-Za-z]*)", "$1");
+            instigatorString = instigatorString.replaceAll("([A-Z])", " $1");
```

Comment:
Could you shortly explain what the format of the `parentPrefab.getName()` is, and what the `replaceAll` calls do?